aristanetworks/ansible-avd
# refactor eos_cli_config_gen router bfd

## Change Summary

## Types of changes


## Related Issue(s)

resolves issue #713 

## Component(s) name

`arsita.avd.eos_cli_config_gen`
`arista.avd.eos_designs`

## Proposed changes

Refactor router bfd to confirm to data model standards

Current data model:

```yaml
bfd_multihop:
  interval: < rate in milliseconds >
  min_rx: < rate in milliseconds >
  multiplier: < 3-50 >
```

Proposed data model:

```yaml
router_bfd:
  multihop:
    interval: < rate in milliseconds >
    min_rx: < rate in milliseconds >
    multiplier: < 3-50 >
```

## How to test

Verify molecule results, no change to eos configuration for the exception of l2leafs, which removes `router bfd` configuration that was not required.

## Checklist:
